fixtures/composed-class-names/fromfile.css

```diff
@@ -0,0 +1 @@
+.from-class {}
```

Comment:
```suggestion
.from-class {}

```

Just a tiny suggestion to see if CircleCI will run the test for forked pull requests.